### PR TITLE
Release taxcalc 3.4.1

### PR DESCRIPTION
**Pull request by jdebacker**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jdebacker** [comment]
@conda-forge-admin, please rerender

**Comment by jdebacker** [comment]
@conda-forge/core the checks on Azure pipeline are failing with a message that is not very helpful:

```
TESTS FAILED: taxcalc-3.4.1-pyhd8ed1ab_0.conda
##[error]Bash exited with code '1'.
```

All unit tests for the package itself are passing.  What are the test failures referred to here?

**Comment by hmaarrfk** [comment]
Conda tests are defined in https://github.com/conda-forge/taxcalc-feedstock/blob/main/recipe/meta.yaml#L37

I added a missing dependency, please check the minimum requirements for it

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,6 +30,7 @@ requirements:
     - numba
     - paramtools >=0.18.0
     - aiohttp
+    - pkg_resources
```
Comment (hmaarrfk) [inline recipe/meta.yaml:33]:
```suggestion
    - setuptools
```

**Comment by jdebacker** [comment]
[@conda-forge-admin](https://github.com/conda-forge-admin) please rerun Azure Pipelines checks

**Comment by hmaarrfk** [comment]
The command tc test is failing for a real reason: 
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=733562&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=589


Admin commands are documented in: https://conda-forge.org/docs/maintainer/infrastructure.html#admin-web-services

**Comment by jdebacker** [comment]
@hmaarrfk Thank you for your help!  I see the test failure and although I don't get that locally, I've made a change to the underlying package that I *think* should fix that.  I've published a new release and hope that when the Azure pipeline checks run again this will work.

**Pull request merged**